server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -98,7 +99,9 @@ public class FollowersChecker {
         "cluster.fault_detection.follower_check.timeout",
         TimeValue.timeValueMillis(10000),
         TimeValue.timeValueMillis(1),
-        Setting.Property.NodeScope
+        TimeValue.timeValueMillis(60000),
```

Comment:
Are we keeping this 60s to match the `JOIN_TIMEOUT_SETTING`? Trying to understand about the timevalue of 60s here

Reply:
This value is independent of `JOIN_TIMEOUT_SETTING`. The default value is still 10s for individual leader/follower check timeout. Since we are making it dynamic now, putting 60s as maximum to prevent users from setting arbitrarily large values. With 60s, and default 3 retries, we should get 3m for node to respond before it is kicked out from the cluster. Does that sound reasonable?

Reply:
That makes sense. Thanks @jainankitk!